app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :increment_no_of_comments
+
+  private
+
+  def increment_no_of_comments
+    post.update(comments_counter: post.comments_counter + 1)
+  end
```

Comment:
- [ ] As I pointed out in the `Post` model, kindly use the same suggestion to use `increment!` built in method 👍